scripts/install.sh

```diff
@@ -0,0 +1,368 @@
+#!/bin/sh
+set -e
+
+usage() {
+  this=$1
+  cat <<EOF
+$this: download go binaries for trufflesecurity/trufflehog
+
+Usage: $this [-b] bindir [-d] [tag]
+  -b sets bindir or installation directory, Defaults to ./bin
+  -d turns on debug logging
+   [tag] is a tag from
+   https://github.com/trufflesecurity/trufflehog/releases
+   If tag is missing, then the latest will be used.
+
+EOF
+  exit 2
+}
+
+parse_args() {
+  #BINDIR is ./bin unless set be ENV
+  # over-ridden by flag below
+
+  BINDIR=${BINDIR:-./bin}
+  while getopts "b:dh?x" arg; do
+    case "$arg" in
+      b) BINDIR="$OPTARG" ;;
+      d) log_set_priority 10 ;;
+      h | \?) usage "$0" ;;
+      x) set -x ;;
+    esac
+  done
```

Comment:
My second request for long options :)

I know `getopt` supports it but not sure about `getopts`. What do you think about supporting long options in the installation script?

Reply:
getops does not support long options, its limited to single character options only. There are workaround for getopts to support long options but its not error proof requiring frequent maintenance. Other option is to switch over to getopt. I'm not sure how frequently long options are requested. 

Please let me know if you think long options are necessary. Happy to change it.

Reply:
Hm, I think in this case the short options are fine. The install script will most often be used in a copy-paste situation and there aren't too many of them.